### ISSUE TITLE
[Concepts -> Source Chains] Some zome fns seem to not work

### ISSUE THREAD

**Issue by jost-s** [holochain-gym/holochain-gym.github.io#42]
In the playground of the Source Chains chapter there are 3 tabs with zome fns.

- On "Demo_Entries" the only two that have an effect for me are "create_entry" and "delete_entry". For the other ones nothing happens when I enter a hash of an entry and click "Execute".
- On "Demo_Links" the function "get_links" has no effect either

Am I using these functions wrong? Or what should happen when I execute "get" or "get_details" for example?

**Comment by guillemcordoba** [comment]
Oh right... I think we are missing the results panel here. The reason they do nothing is 1) because they are a readonly query (get, query, get_links) and as such they don't modify the source-chain, or 2) that it's not clear what the input should be (entryhash vs headerhash, etc.). 

Basically you need feedback of the result of executing the function, which exists in other pages, it's just a matter of including it here. 

Thanks :)

**Comment by jost-s** [comment]
Yes, I agree, I was expecting to see the results of a `get` function somewhere. The input was clear to me.

**Comment by guillemcordoba** [comment]
Cool cool
